lib/network/probe/ProbeServer.js

```diff
@@ -0,0 +1,97 @@
+const httpServer = require('../http/server');
```

Comment:
Can you rename the file to `HealthProbeServer.js` so that it has the same name than the class?